### PR TITLE
Fix can't build on ios because of unknown type name `mixpanel`

### PR DESCRIPTION
**Pull request by Kabangi**

### CODE REVIEW

**Comment by Gadigg** [comment]
Can someone merge this pull request? 
This keeps the package from integrating with CI platforms...

File: RNMixpanel/RNMixpanel.m [hunk 1]
```diff
@@ -46,18 +46,19 @@ -(Mixpanel*) getInstance: (NSString *)name {
...
 }
 
+// setAppSessionProperties iOS
 // setAppSessionProperties iOS
 RCT_EXPORT_METHOD(setAppSessionPropertiesIOS:(NSDictionary *)properties) {
     if ([properties objectForKey:@"minimumSessionDuration"]) {
         NSNumber *minimumSessionDuration = properties[@"minimumSessionDuration"];
         long long int intValue = [minimumSessionDuration longLongValue];
-        mixpanel.minimumSessionDuration = intValue;
+        [Mixpanel sharedInstance].maximumSessionDuration = intValue;
```
Comment (JoelHensley) [inline RNMixpanel/RNMixpanel.m]: Shouldn't this be `minimumSessionDuration`?

Comment (muhammadsr) [reply RNMixpanel/RNMixpanel.m]: Yep!

Comment (Kabangi) [reply RNMixpanel/RNMixpanel.m]: Sending an update to affect this on this PR

**Comment by lorenjohnson** [comment]
Awesome. Looks like this is ready to go, thank you @Kabangi. Hopefully it'll get merged soon otherwise I'll figure out how to temporarily install the package from this branch...

File: RNMixpanel/RNMixpanel.m [hunk 2]
```diff
@@ -46,18 +46,19 @@ -(Mixpanel*) getInstance: (NSString *)name {
     }
 }
 
+// setAppSessionProperties iOS
```
Comment (lorenjohnson) [inline RNMixpanel/RNMixpanel.m]: extra comment can be removed

**Comment by davodesign84** [comment]
Merged and published to npm :)

**Pull request merged**